.travis.yml

```diff
@@ -44,6 +44,5 @@ jobs:
         - chmod +x Build/travis-push.sh
       script:
         - pwsh -File './Build/build.ps1'
-        - rm $PSDS_CODECOVERAGE_RESULTFILE
```

Comment:
The file gets removed in the Set-PSDesiredStateConfigurationTestsResults function on line 448. This is here because the file does not get created if the unit tests for the VMware.PSDesiredStateConfiguration never get run.